irohad/consensus/yac/yac.hpp

```diff
@@ -18,6 +18,7 @@
 #include "consensus/yac/outcome_messages.hpp"  // because messages passed by value
 #include "consensus/yac/storage/yac_vote_storage.hpp"  // for VoteStorage
 #include "logger/logger_fwd.hpp"
+#include "main/subscription.hpp"
 
 #include <rxcpp/operators/rx-observe_on.hpp>
```

Comment:
remove this rx-cpp includes and from vcpkg

Reply:
https://imgur.com/3l4KntA.png